setup.cfg

```diff
@@ -126,6 +126,105 @@ images =
     accelerate~=0.23.0  # For the newer versions of Transformers
     pillow~=9.4.0
 
+heim =
```

Comment:
Are all of these needed?

In general, this list should only be _direct_ dependencies, not _transitive_ dependencies.